apps/meteor/client/hooks/useContinuousSoundNotification.ts

```diff
@@ -0,0 +1,53 @@
+import type { ICustomSound } from '@rocket.chat/core-typings';
+import { useSetting, useUserPreference, useUserSubscriptions } from '@rocket.chat/ui-contexts';
+import { useEffect } from 'react';
+
+import { CustomSounds } from '../../app/custom-sounds/client/lib/CustomSounds';
+
+const query = { t: 'l', ls: { $exists: false }, open: true };
+export const useContinuousSoundNotification = () => {
+	const userSubscriptions = useUserSubscriptions(query);
+
+	const playNewRoomSoundContinuously = useSetting('Livechat_continuous_sound_notification_new_livechat_room');
+
+	const newRoomNotification = useUserPreference<string>('newRoomNotification') || '';
+	const audioVolume = useUserPreference<number>('notificationsSoundVolume');
+
+	const continuousCustomSoundId = `${newRoomNotification}-continuous`;
+
+	const volume = audioVolume ? Number((audioVolume / 100).toPrecision(2)) : 1;
```

Comment:
I know this is a great improvement but if a user sets its volume to 1 that means he/she wont listen anything (or super low), which defeats the purpose of the "continuous sound notifs" setting 🤔 

Can we check if that new behavior is compatible with the setting's expected behavior?

If that's good, then consider that if a user sets his volume as 0, this condition will play it at full sound cause `0 ?` will be false

Reply:
We should be good, as the message notification sounds on omnnichannel use the volume setting, as for the `0 ?` issue, I've fixed that on a subsequent commit